test/test_type.jl

```diff
@@ -0,0 +1,98 @@
+module TestType
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+# Start with a clean environment: remove Trixi.jl output directory if it exists
+outdir = "out"
+isdir(outdir) && rm(outdir, recursive = true)
+
+# Run unit tests for various equations
+@testset "Float32 Type Stability" begin
+    @timed_testset "Acoustic Perturbation 2D" begin
+        v_mean_global = (0.0f0, 0.0f0)
```

Comment:
I think it would be nice to test type stability here for both `Float32` and `Float64`, e.g., 
```julia
for RealT in (Float32, Float64)
    v_mean_global = (zero(RealT), zero(RealT))
    ...
end
```